package.json

```diff
@@ -1,91 +1,91 @@
-        "@types/supertest": "^2.0.8",
-        "audit-filter": "^0.5.0",
-        "aws-lambda-mock-context": "^3.2.1",
-        "babel-jest": "^24.9.0",
-        "gulp": "^4.0.2",
-        "husky": "^3.0.9",
-        "jest": "^24.9.0",
-        "jest-plugin-context": "^2.9.0",
-        "jest-sonar-reporter": "^2.0.0",
-        "lambda-tester": "^3.5.0",
-        "lodash": "^4.17.19",
-        "serverless": "^1.56.1",
-        "serverless-dynamodb-local": "^0.2.37",
-        "serverless-offline": "^5.12.0",
-        "serverless-plugin-tracing": "^2.0.0",
-        "serverless-plugin-typescript": "^1.1.9",
-        "supertest": "^4.0.2",
-        "ts-jest": "^24.1.0",
-        "ts-node": "^8.4.1",
-        "tslint": "^5.20.0",
-        "tslint-jsdoc-rules": "^0.2.0",
-        "typescript": "^3.6.4"
-    },
-    "dependencies": {
-        "aws-lambda": "^0.1.2",
-        "aws-sdk": "^2.561.0",
-        "aws-xray-sdk": "^2.4.0",
-        "node-yaml": "^3.2.0",
-        "path-parser": "^4.2.0",
-        "reflect-metadata": "^0.1.13"
-    },
-    "jestSonar": {
-        "reportPath": ".reports",
-        "reportFile": "test-report.xml",
-        "indent": 4
-    },
-    "jest": {
-        "coverageDirectory": "./coverage",
-        "collectCoverage": true,
-        "testResultsProcessor": "jest-sonar-reporter",
-        "testURL": "http://localhost"
-    },
-    "husky": {
-        "hooks": {
-            "pre-commit": "npm run prepush && npm run security-checks",
-            "pre-push": "npm run prepush"
-        }
+  "name": "cvs-svc-defects",
+  "version": "1.0.0",
+  "description": "Microservice that stores and serves the defects data",
+  "main": "src/handler.js",
+  "engines": {
+    "node": "10.18.1",
+    "npm": "5.6.0"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dvsa/cvs-svc-defects.git"
+  },
+  "scripts": {
+    "start": "BRANCH=local SLS_DEBUG=* ./node_modules/serverless/bin/serverless offline start",
+    "startx": "npm run tslint && BRANCH=local SLS_DEBUG=* ./node_modules/serverless/bin/serverless offline start",
+    "test": "BRANCH=local SLS_DEBUG=* jest --testMatch=\"**/*.unitTest.ts\" --coverage",
+    "build": "node_modules/typescript/bin/tsc --rootDir ./ --outDir .build --sourceMap false && npm run build:copy",
+    "build:copy": "find src -type f \\( -name \"*.yml\" -or -name \"*.json\" \\) | cpio -pdm .build && find tests -type f \\( -name \"*.yml\" -or -name \"*.json\" \\)  | cpio -pdm .build",
+    "test-i": "BRANCH=local jest --testMatch=\"**/*.intTest.ts\" --runInBand",
+    "end:test": "node_modules/gulp/bin/gulp.js kill-serverless",
+    "prepush": "BRANCH=local npm test && npm run tslint",
+    "predeploy": "npm install && npm run prepush",
+    "security-checks": "git secrets --scan && git log -p | scanrepo",
+    "tslint": "tslint src/**/*.ts tests/**/*.ts --out .reports/lint_issues.json",
+    "sonar-scanner": "npm run tslint && npm run coverage && sonar-scanner",
```

Comment:
suggest to update this line to match to test-results. This is to avoid duplicate calls to integration tests.

Reply:
good spot thanks, also updated the prepush, feel free to resolve if happy